ranch/priority_test.go

```diff
@@ -32,15 +32,15 @@ func TestRequestQueue(t *testing.T) {
 	rq := newRequestQueue()
 	count := 10
 	for i := 0; i < count; i++ {
-		rank, new := rq.getRank(fmt.Sprintf("request_%d", i), testTTL, now)
+		rank, new, _ := rq.getRank(fmt.Sprintf("request_%d", i), testTTL, now)
```

Comment:
it'd be good to have at least one of these test cases cover the new functionality.